profiles/package.mask

```diff
@@ -1,3 +1,9 @@
+# Wolfgang E. Sanyer <WolfgangESanyer@gmail.com> (2021-02-11)
```

Comment:
I suggest pushing these commits right now outside this pull request.

Reply:
good idea, I'll do that.